images/scripts/rhel.setup

```diff
@@ -304,13 +310,33 @@ if [ "$DO_BUILD" -eq 1 ]; then
     # install mock from EPEL
     major=$(cut -d '-' -f 2 <<< "$IMAGE")
     minor=$(cut -d '-' -f 3 <<< "$IMAGE")
-    dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${major}.noarch.rpm
-    dnf install -y rpm-build mock
-    dnf config-manager --set-disabled epel
+    dnf install -y rpm-build
+
+    if [ "${IMAGE#rhel-10*}" != "$IMAGE" ] || [ "${IMAGE#centos-10}" != "$IMAGE" ]; then
+        # RHEL 10 has no EPEL yet, install from Fedora 40
```

Comment:
Ugh....

*looks the other way*

Reply:
We did the same when introducing RHEL 9 back then, see commit 6d09ffa29253c6ec33a5ea1f9b644ce7f5b759bb